src/utils/pages.ts

```diff
@@ -15,12 +15,13 @@ limitations under the License.
 */
 
 import { logger } from "matrix-js-sdk/src/logger";
+import { MatrixClient } from "matrix-js-sdk/src/matrix";
 
 import { IConfigOptions } from "../IConfigOptions";
 import { getEmbeddedPagesWellKnown } from "../utils/WellKnownUtils";
 import { SnakedObject } from "./SnakedObject";
 
-export function getHomePageUrl(appConfig: IConfigOptions): string | undefined {
+export function getHomePageUrl(appConfig: IConfigOptions, matrixClient: MatrixClient | undefined): string | undefined {
```

Comment:
I think it might be worth adding a note about why it's expected for the client to be undefined where this util is used

Reply:
This was me just specifying the loosest allowable type, even if it didn't make sense, ditched `unknown` - thanks